docs/_newsfragments/1902.rst

```diff
@@ -0,0 +1,2 @@
+Re-add ``api_helpers`` module that was renamed ``app_helpers``.
+This modules is to consider deprecated and will be removed in a future falcon version.
```

Comment:
> _This modules (sic) is to consider (sic) deprecated and will be removed in a future falcon version._

This sentence could be cleaned up a bit... maybe something along the lines of :arrow_down: 
> This module is considered deprecated, and will be removed in a future Falcon version.